byterun/caml/spacetime.h

```diff
@@ -131,12 +133,17 @@ typedef struct {
 } allocation_point;
 
 typedef struct {
+  value callee_node;
+  value call_count;
```

Comment:
Should this be guarded by `WITH_SPACETIME_CALL_COUNTS`? It is used in `spacetime.c` which suggests it should, but it is also used in spacetime_offline.c` which suggests it shouldn't.